crates/daphne-server/src/metrics.rs

```diff
@@ -76,6 +78,10 @@ mod prometheus {
         fn daphne(&self) -> &dyn DaphneMetrics {
             self
         }
+
+        fn aggregate_job_latency(&self, _time: Duration) {
+            // unimplemented by default
+        }
```

Comment:
Why is this unimplemented by default?

Reply:
Because it's higher cardinality and I'm not sure who all wants to use it.

Reply:
Alright, leave a comment in the code with this justification and we can merge